release-team/role-handbooks/docs/README.md

```diff
@@ -232,36 +232,53 @@ The middle weeks of the launch are where the Docs Lead and Docs Lead Shadows tra
+    ```bash
+    # Step 0 (if you don't already have a remote called "upstream")
+    git remote add upstream https://github.com/kubernetes/website.git
+    # Step 1
+    git fetch upstream
+    # Step 2
+    git checkout upstream/release-[current release]
+    # Step 3
+    git merge upstream/master
+    # Set the commit message to, eg,
+    # "Merged master into release-[current release] to keep in sync"
+    # Step 4
+    git checkout -b merged-master-release-[current release]
+    # Step 5
+    git push origin merged-master-release-[current release]
     ```
 
     Submit a PR against upstream ```release-[current release]```  from your fork  ```merged-master-release-[current release]``` branch with label ```tide/merge-method-merge```. e.g [merge master into current release](https://github.com/kubernetes/website/pull/16224)
 
+    <hr />
+
     To merge `master` into `dev-[future release]` on your local fork:
 
-    ```
-    $ git remote add upstream https://github.com/kubernetes/website.git
-    $ git fetch upstream
-    $ git checkout upstream/dev-[future release]
-    $ git merge upstream/master
-    $ git checkout -b merged-master-dev-[future release]
-    $ git commit -m "merged master into dev-[future release] to keep in sync"
+    ```bash
+    # Step 0 (if you don't already have a remote called "upstream")
+    git remote add upstream https://github.com/kubernetes/website.git
+    # Step 1
+    git fetch upstream master
+    # Step 2
+    git checkout upstream/dev-[future release]
+    # Step 3. You might see merge conflicts at this point.
+    git merge upstream/master
     ## if needed: https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/
     ## git add ...
-    ## git commit -m "resolving conflicts"
-    $ git push origin merged-master-dev-[future release]
+    ## git merge --continue
+    # Step 4
+    git checkout -b merged-master-dev-[future release]
+    # Step 5
+    git push origin merged-master-dev-[future release]
     ```
 
-    Submit a PR against upstream ```dev-[future release]```  from your fork  ```merged-master-dev-[future release]``` branch with label ```tide/merge-method-merge```. e.g [merge master into future release](https://github.com/kubernetes/website/pull/16225)
-
     You may need to fix conflicts manually. If somebody has improved a page on master, and at the same time it has been updated in dev-1.14, we may need to figure out how to make those changes work together. If something comes up which isn't obvious, you can always abort the merge and reach out to SIG Docs for help.
+    When you have completed resolving the differences manually, run `git merge --continue` to complete the merge. Then carry on from step 4.
+
+    Submit a PR against upstream ```dev-[future release]```  from your fork  ```merged-master-dev-[future release]``` branch with label ```tide/merge-method-rebase```. e.g [merge master into future release](https://github.com/kubernetes/website/pull/16225)
```

Comment:
```suggestion
    Submit a PR against upstream `dev-[future release]` from your fork `merged-master-dev-[future release]` branch with label `tide/merge-method-rebase`. e.g., [merge master into future release](https://github.com/kubernetes/website/pull/16225)
```

Reply:
@zacharysarah -- Same here. Any reason to use the rebase merge method here?

---

@sftim -- the referenced PR uses the `merge` merge method, not `rebase`.

Reply:
No longer relevant